# control the logger
#if true the logger is enabled
bool enable
# log level, must be greater than 0 to enable logging
uint8 level
#name prefixe of log
string name